frontend/src/components/inbox/components/Chat.tsx

```diff
@@ -114,12 +114,6 @@ export function Chat() {
                     ? [
                         <Avatar
                           key={message.id}
-                          title={`${subscriber.first_name} ${
-                            subscriber.last_name
-                          } : ${normalizeDate(
-                            i18n.language,
-                            message.createdAt,
-                          )}`}
```

Comment:
I suggest keeping the name and removing only the date : 
```suggestion
                         title={`${subscriber.first_name} ${subscriber.last_name}`}
```